readme.md

```diff
@@ -320,6 +320,10 @@ To run **remark**, optionally with **remark-lint** from **Gulp**, use
 
 To check markdown documents editing with Vim, use [**ale**][ale].
 
+For [**Sublime Text**](https://www.sublimetext.com) use [**SublimeLinter-contrib-remark-lint**](https://packagecontrol.io/packages/SublimeLinter-contrib-remark-lint) — part of [**SublimeLinter**](http://www.sublimelinter.com/en/stable/) framework.
+
+[**Grunt**](https://gruntjs.com/) users can use [**grunt-remark**](https://www.npmjs.com/package/grunt-remark).
```

Comment:
Grunt is more a build tool than an editor.
Maybe add in a new section for build too integrations?

```suggestion
## Build Tool Integrations

[**Grunt**](https://gruntjs.com/) users can use [**grunt-remark**](https://www.npmjs.com/package/grunt-remark).
```

Reply:
@ChristianMurphy, personally, I would make a common table without extra words; example:

| [**Sublime Text**](https://www.sublimetext.com) | [**SublimeLinter-contrib-remark**](https://packagecontrol.io/packages/SublimeLinter-contrib-remark-lint) |
|-------------------------------------------------|----------------------------------------------------------------------------------------------------------|
| [**Grunt**](https://gruntjs.com/)               | [**grunt-remark**](https://github.com/remarkjs/grunt-remark)                                             |

But it does not matter to me; do as you wish.

Thanks.

Reply:
My thought would be to keep the format as is to be consistent with other remark/unified plugins.
@Murderlon @wooorm thoughts? :thought_balloon:

Reply:
@Kristinita Also nice!

@ChristianMurphy Hmm, Gulp is in the first section, so I think splitting this up makes more sense in a separate PR! I’d personally go with a list of things, just like most other lists of plugins/related projects, e.g.:

```markdown
## Integrations

*   [linter-markdown](#) ([Atom](#)) — use remark-lint from Atom
*   [linter-remark](#) ([Atom](#)) — use all of remark from Atom
*   [SublimeLinter-contrib-remark-lint](#) ([Sublime](#)) — use remark-lint from Sublime Text
*   [ale](#) ([Vim](#)) — use remark-lint from Vim
*   [gulp-remark](#) ([Gulp](#)) — use all of remark with Gulp
*   [grunt-remark](#) ([Grunt](#)) — use all of remark with Grunt

We’re interested in more integrations. Let us know if we can help.
```

»

## Integrations

*   [linter-markdown](#) ([Atom](#)) — use remark-lint from Atom
*   [linter-remark](#) ([Atom](#)) — use all of remark from Atom
*   [SublimeLinter-contrib-remark-lint](#) ([Sublime](#)) — use remark-lint from Sublime Text
*   [ale](#) ([Vim](#)) — use remark-lint from Vim
*   [gulp-remark](#) ([Gulp](#)) — use all of remark with Gulp
*   [grunt-remark](#) ([Grunt](#)) — use all of remark with Grunt

We’re interested in more integrations. Let us know if we can help.